Dockerfile-GPU

```diff
@@ -11,7 +11,7 @@ RUN mkdir -p /home/user/file-upload && chmod 777 /home/user/file-upload
 # Install software dependencies
 RUN apt-get update && apt-get install -y software-properties-common && \
     add-apt-repository ppa:deadsnakes/ppa && \
-    apt-get update && apt-get install -y \
```

Comment:
This `apt-get update` might be necessary to be able to get packages from the PPA that was added just above, but I'm not 100% sure, I'm going to try it out

Reply:
No problem, seems to be working fine